web-ui/src/components/volunteer/VolunteerTables.css

```diff
@@ -30,31 +47,46 @@
   td,
   th {
     padding: 0.5rem;
+    color: black; /* Ensures text color remains black */
```

Comment:
This is actually not the right color. This should be: `var(--checkins-palette-text-primary)`